src/utils/DateUtils.ts

```diff
@@ -142,25 +306,28 @@ export class DateUtils {
     }
   }
 
+  /**
+   * Converts a month from a number to a string (e.g '1' to 'january').
+   * @param month The month represented by it's number (1 to 12).
+   * @returns {string} The month's name associated to it's number.
+   */
   static monthToString(month: number): string {
-    var date = new Date(1980, month);
-    return Globalize.format(date, 'MMMM');
+    DateUtils.setLocale();
+    const date = moment(new Date(1980, month)).toDate();
+    return moment(date).format('MMMM');
   }
 
+  /**
+   * Checks if the input is an instance of Date.
+   * @param date The value to evaluate.
+   * @returns {boolean} False if the result is not an instance of Date.
+   */
   static isValid(date: any) {
+    DateUtils.setLocale();
+
     if (date instanceof Date) {
       return !isNaN(date.getTime());
     }
     return false;
   }
-
-  static timeBetween(from: Date, to: Date) {
```

Comment:
This is a public method. I don't know if there was a discussion about this but be aware that this is a breaking change.